CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/templates/SimpleQueryPlugin.java

```diff
@@ -171,10 +171,10 @@ public final void run(final PluginGraphs graphs, final PluginInteraction interac
                         interaction.notify(PluginNotificationLevel.INFO, "Action cancelled: " + graphs.getGraph() + ": " + getName());
                         throw e;
                     } catch (Exception e) {
-                        final String msg = Bundle.MSG_Query_Failed(graph, getName());
-                        interaction.notify(PluginNotificationLevel.ERROR, msg + "\n" + e.getMessage());
                         cancelled = true;
-                        LOGGER.log(Level.WARNING, msg, e);
+                        final String msg = Bundle.MSG_Query_Failed(graph, getName());
+                        interaction.notify(PluginNotificationLevel.INFO, msg + "\n" + e.getMessage());
```

Comment:
not sure why this is downgraded to INFO @serpens24 ?

Reply:
ok I caught up on the discussion on a different thread and understand why you made it INFO. So the `LOGGER.log` is going to log it so if `interaction.notify()` is to log it then do we need it?

what's your thoughts on this @cygnus-x-1 ?

Reply:
My thinking was I could eiter delete it, or downgrade it. I opted to downgrade, but honestly its a coinflip IMO.

Reply:
I've had another look at all of the logs and here is a recap based on your branch:

* INFO message that runs on every data access plugin - OK
```
INFO [au.gov.asd.tac.constellation.views.dataaccess.panes.DataAccessPane]: 		Running Select Top N
```

* FINE log for any plugin - OK and can be turned off if we don't want to see FINE messages
```
FINE [au.gov.asd.tac.constellation.plugins.PluginExecution]: Select Top N
```

* the downgraded INFO message which I think we can drop all together @serpens24 ?
```
INFO [au.gov.asd.tac.constellation.views.dataaccess.panes.DataAccessPane]: Plugins run.
Select Top N: Action failed: au.gov.asd.tac.constellation.graph.locking.DualGraph@42945620; Select Top N
Invalid mode value provided
```

* a WARNING level message - OK
```
WARNING [au.gov.asd.tac.constellation.plugins.templates.SimpleQueryPlugin]: Action failed: au.gov.asd.tac.constellation.graph.locking.DualGraph@42945620; Select Top N
```

* a SEVERE version probably because of the `throw e` if I understand it
```
SEVERE [au.gov.asd.tac.constellation.graph.node.plugins.DefaultPluginInteraction]: Select Top N: Invalid mode value provided
au.gov.asd.tac.constellation.plugins.PluginException: Invalid mode value provided
	at au.gov.asd.tac.constellation.views.dataaccess.plugins.utility.SelectTopNPlugin.edit(SelectTopNPlugin.java:202)
	at au.gov.asd.tac.constellation.plugins.templates.SimpleQueryPlugin.describedEdit(SimpleQueryPlugin.java:265)
	at au.gov.asd.tac.constellation.plugins.templates.SimpleQueryPlugin.run(SimpleQueryPlugin.java:165)
	at au.gov.asd.tac.constellation.graph.node.plugins.DefaultPluginEnvironment.lambda$executePluginLater$0(DefaultPluginEnvironment.java:123)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

what do you think @serpens24  and anyone else?

Reply:
I'm happy to drop the dowgraded error .... will now.